website/server/controllers/api-v3/members.js

```diff
@@ -384,18 +384,27 @@ api.transferGems = {
     let promises = [receiver.save(), sender.save()];
     await Bluebird.all(promises);
 
-    let message = res.t('privateMessageGiftIntro', {
-      receiverName: receiver.profile.name,
-      senderName: sender.profile.name,
+    // generate the message in both languages, so both users can understand it
+    let receiverLang = receiver.preferences.language;
+    let senderLang = sender.preferences.language;
+    let [receiverMsg, senderMsg] = [receiverLang, senderLang].map((lang) => {
+      let messageContent = res.t('privateMessageGiftIntro', {
+        receiverName: receiver.profile.name,
+        senderName: sender.profile.name,
+      }, lang);
+      messageContent += res.t('privateMessageGiftGemsMessage', {gemAmount}, lang);
+      messageContent = `\`${messageContent}\` `;
```

Comment:
It preserves the current behaviour, but sure, could be changed. I don't think it would be any nicer if moved below though. What would do it in general would be a utility function to wrap strings in code tags, so such ugly packaging can be hidden behind an interface. It would make changing them later a bit easier too (eg. for party chat).